package.json

```diff
@@ -31,9 +31,12 @@
     "tap-spec": "^4.1.1"
   },
   "dependencies": {
+    "are-we-there-yet": "^1.1.4",
     "body-parser": "^1.17.2",
     "env2": "^2.2.0",
     "express": "^4.15.3",
-    "pg": "^6.4.0"
+    "pg": "^6.4.0",
+    "signal-exit": "^3.0.2",
+    "wide-align": "^1.1.2"
```

Comment:
We tried to get rid of all these unnecessary dependencies, but the server won't start without them.. we left them in our most recent PR..